audit/result.go

```diff
@@ -142,7 +142,10 @@ func (r *Result) ForContainer(cnr cid.ID) {
 // Zero Result has nil key. Return value MUST NOT be mutated: to do this,
 // first make a copy.
 //
-// See also SetAuditorPublicKey.
+// Be noticed, this is a compressed public key. See [elliptic.MarshalCompressed].
+// Use [neofsecdsa.PublicKey.Decode] to get public key back.
```

Comment:
`... to decode it into a type-specific structure`?

Reply:
Updated